flightgear.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-FG_ROOT=/app/share/flightgear FG_AIRCRAFT=$XDG_DATA_HOME/fgfs/Aircraft/ FG_SCENERY=$XDG_DATA_HOME/fgfs/Scenery fgfs --launcher "$@"
```

Comment:
Why did this need to change?

Reply:
Hi, I'm interested in getting this merged so I'll answer.

Basically, the fgfs directory never existed in the xdg-data directory, but `~/.fgfs` exists both in the sandbox and on the host, so those changes are good. Can this MR be merged @TingPing ? It would allow downloading the scenery and all the extra aircrafts, which isn't currently possible.

Reply:
I'll merge it if the build passes. Might need updating.

Reply:
GLEW is missing, as of build logs `Could NOT find GLEW (missing: GLEW_INCLUDE_DIR GLEW_LIBRARY)`. I can't commit to this branch but the missing thing to fix this is to add `- shared-modules/glew/glew.json` in the `modules` list.

Reply:
Hi @Vanadiae, thank you for refreshing this issue. I pushed the modification you suggested

Reply:
Ok, so now it fails because of this message, `Could NOT find GLUT (missing: GLUT_glut_LIBRARY GLUT_INCLUDE_DIR)`, so we need to add FreeGLUT (a free software implementation for GLUT). Here's what need to be inserted in the modules list (before the openscenegraph module will probably work).:

```
- name: freeglut
  buildsystem: cmake
  config-opts:
  - -DCMAKE_BUILD_TYPE=Release
  - -DFREEGLUT_BUILD_DEMOS=OFF
  - -DFREEGLUT_BUILD_SHARED_LIBS=ON
  sources:
  - type: archive
    url: https://sourceforge.net/projects/freeglut/files/freeglut/3.2.1/freeglut-3.2.1.tar.gz
    sha256: bc606a149d3ba084f7ed78f07d62eb86ee5e586c61cfa6d1ad37009283b4de8f
```

Care must be taken to respect the same indentation as for the other modules (basically keep the indentation as in the code block just before, and indent this with two more spaces).